python-sdk/src/astro/databases/google/bigquery.py

```diff
@@ -73,6 +73,15 @@ class BigqueryDatabase(BaseDatabase):
         FileLocation.S3: "load_s3_file_to_table",
         FileLocation.LOCAL: "load_local_file_to_table",
     }
+    AUTODETECT_SCHEMA_SUPPORTED: set[FileLocation] = {
```

Comment:
@sunank200  I think there is some conflict between this implementation and https://github.com/astronomer/astro-sdk/pull/780/files for the bigquery. Can we check this?

Reply:
I am have merged that code to my branch and its running fine.